internal/monitors/cadvisor/kubeletstats.go

```diff
@@ -29,6 +30,8 @@ type KubeletStatsConfig struct {
 	config.MonitorConfig
 	// Kubelet client configuration
 	KubeletAPI kubelet.APIConfig `yaml:"kubeletAPI" default:""`
+	// Whether or not to collect ephemeral storage stats from pods
+	IncludePodEphemeralStats bool `yaml:"includePodEphemeralStats" default:"false"`
```

Comment:
The whole point of doing the group thing was so that we don't need flags like this.  The use just has to specify `extraGroups: [podEphemeralStats]` and you can check it using `output.HasEnabledMetricInGroup(podEphemeralStats)`.  This will capture both if they use `extraGroups` or `extraMetrics`.

Get rid of this config.

Reply:
Done, removed the config. I hadn't realized the example I followed was more to maintain compatibility with old stuff.